### PR TITLE
added OIN token asset (sei)

### PR DESCRIPTION
**Pull request by Dizz02**

### CODE REVIEW

File: sei/assetlist.json [hunk 1]
```diff
@@ -23,6 +23,25 @@
...
             },
             "coingecko_id": "sei-network"
+        },
+
+        {
+            "description": "OIN Token ($OIN) is a pioneering digital asset developed on the $SEI Blockchain. Representing more than just a cryptocurrency, $OIN is a symbol of the future of decentralized finance, designed with the vision of reshaping the crypto landscape.",
+            "denom_units": [
+                {
+                    "denom": "factory/sei12q0zv3c4cd9jkupn0krazdycc5ftw9wzt9vmhu/OIN",
+                    "exponent": 6
```
Comment (JeremyParish69) [inline sei/assetlist.json:33]: exponent for base denom must be 0.

File: sei/assetlist.json [hunk 2]
```diff
@@ -23,6 +23,25 @@
...
+                }
+            ],
+            "base": "OIN",
+            "name": "OIN",
+            "display": "oin",
```
Comment (JeremyParish69) [inline sei/assetlist.json:38]: this demon isn't defined. You can add another denom_unit above called 'oin' with exponent 6.

File: sei/assetlist.json (hunk 1 above)
Comment (Dizz02) [reply sei/assetlist.json:37]: Done thanks

File: sei/assetlist.json (hunk 2 above)
Comment (Dizz02) [reply sei/assetlist.json:43]: Done too

**Pull request merged**